### ISSUE TITLE
Tray popups becoming annoying after update KB5022913 (Win11)

### ISSUE THREAD

**Issue by TriggerEdge** [winsiderss/systeminformer#1643]
### Description of the feature, modification, idea or suggestion

Is there a way to disable tray popups? 

Previously I just had to click on one of the tray icons to see top processes on each category now they just become "sensitive" to the pointer and appears every time that i need to click on other tray icons.

![image](https://user-images.githubusercontent.com/10005945/224486534-7490f3da-ead2-4dc2-836f-ec062cd79142.png)

**Comment by dmex** [comment]
> Is there a way to disable tray popups?

Options > General > uncheck `Enable Tray Information Window`

> Previously I just had to click on one of the tray icons to see top processes on each category now they just become "sensitive" to the pointer and appears every time that i need to click on other tray icons.

This is entirely controlled by Microsoft and explorer.exe. We have no functionality to change how this works and from 2008 until 2023 from version 1.0 until this week (15 years) the shell notifications were working without a single issue then Microsoft decided to remove them, now they've added them back again and somehow managed to break shell notifications for everyone yet again and because they control the notifications there's nothing anyone can do except disable tray icons until they fix the issue.

**Comment by dmex** [comment]
OK I've come up with a workaround by juggling some timers and pointers to delay the popup for Windows 11... Timers don't have a context so it tries to cache some variables in the global scope that hopefully won't get destroyed and cause access violations.

The tray notification will only be shown when you hover the mouse over the icon for 1 second so it won't be annoying on Win11... If 1 second is too long or not long enough then let me know and I'll change the amount of mouse time required 👍

**Comment by dmex** [comment]
Added with latest nightly build 👍

**Comment by TriggerEdge** [comment]
Hey that works fine now, thank you!

**Comment by rpodric** [comment]
I thought this was my imagination! Thanks for the attention, but maybe the timing (or at least the ability to revert to previous behavior) should be an advanced option, if possible. Just a suggestion.

For me, this program being differently timed than all others on the tray is a bit odd. Not the end of the world, but maybe also not a good default to make it different.

BTW, there's mention of Win11, which I'm also using, but I'm not aware how/why this would be any different in 10.

**Comment by dmex** [comment]
> maybe the timing (or at least the ability to revert to previous behavior) should be an advanced option

The timing could be made customizable but someone needs to make a feature request?

> ability to revert to previous behavior

The previous behavior between 2008 and 2022 worked good but Windows 11 in 2023 ignoring the hover time breaks the tray icons because it shows the menu instantly. If you try to select the icon for third party programs but happen to move the mouse over our icon then Explorer shows our menu over the third party window/menu making it unusable so there needs to be a slight delay.

> I'm not aware how/why this would be any different in 10

Windows 10 correctly shows the menu after 1s hovertime. All of our changes are limited to Windows 11 using a version check. 

TLDR Microsoft blocked/removed most of the shell tray notifications from Windows 11 (open/close/move).... KB5022913 added/fixed the shell tray notifications but they've forgotten to re-include the hover timeout and messages/windows/menus are popping instantly instead of after a delay:

![image](https://user-images.githubusercontent.com/1306177/226304082-d1f4b9a2-126c-4e43-8b0e-6731f4f58262.png)

**Comment by rpodric** [comment]
@dmex OK, I guess I wasn't on 22621.1344 long enough to have picked up on this change, though I did notice right away when it became slower with the later update to SI. I wonder if the "hover treatment" alluded to in KB5022913 is actually working as they intend. With Microsoft, it's often hard to tell the difference between a feature and a bug.

This isn't the only program with a large popup, so those others will continue to act one way (most likely) while System Informer does another.

Thanks

**Comment by rpodric** [comment]
@dmex Do you know if the Microsoft issue still exists now, suggesting that it's probably here for good? If it doesn't, maybe this can be reverted.

**Comment by dmex** [comment]
We added our own hover support, this issue was resolved last year.

**Comment by rpodric** [comment]
@dmex Oh, that's nice. If I wanted the old behavior back, so that it doesn't take hovering for several seconds for it to show up, how would I do that? Not counting outright clicking, since I know that's one way to do it. If it's an Advanced option, I guess it's not using the word "hover,"
